common/djangoapps/third_party_auth/api/tests/test_views.py

```diff
@@ -335,20 +310,6 @@ def test_user_mappings_only_return_requested_idp_mapping_by_provider_id(self):
         self.assertEqual(response.status_code, 200)
         self._verify_response(response, 200, get_mapping_data_by_usernames(LINKED_USERS))
 
-    @ddt.data(
-        (True, True, 200),
-        (False, True, 200),
-        (True, False, 200),
-        (False, False, 401)
-    )
-    @ddt.unpack
-    def test_user_mapping_permission_logic(self, api_key_permission, token_permission, expect):
```

Comment:
Why is this test being removed?

Reply:
this tests logic for access with old permissions. This was replaced by test_list_all_user_mappings_tpa_permission_logic below